src/main/java/io/github/oliviercailloux/y2018/apartments/utils/JsonConvert.java

```diff
@@ -159,20 +115,26 @@ public static Apartment jsonToApartment(Path jsonPath) throws IOException {
 	 * @throws IOException if the JSON file can't be created.
 	 */
 	public static void apartmentsToJson(List<Apartment> listApartments) throws IOException {
```

Comment:
Puisque l'adresse est généré et différentes à chaque lancement du programme, serait-il possible de retourner l'absolute Path qui a été généré. Ainsi l'utilisateur pourra exploiter cette information à sa convenance. (Ex dans de potentielles future exploitation:  Envoi par mail d'une selection ou affichage du chemin du fichier d'export au futur locataire)

Reply:
Fix in commit 98098df77cfa5c4a0b4d9942c0a69dfaa5da918b